### PR TITLE
No Yapf on Contrib folder

### PR DESCRIPTION
**Pull request by lilleswing**
Don't run yapf over the contrib folder.

### CODE REVIEW

File: devtools/travis-ci/test_format_code.sh [hunk 1]
```diff
@@ -1,6 +1,6 @@
 #!/bin/bash -e
 
-CHANGED_FILES=`git diff --name-only $TRAVIS_COMMIT_RANGE | grep .py$`
+CHANGED_FILES=`git diff --name-only $TRAVIS_COMMIT_RANGE | grep .py$ | grep -v contrib/`
```
Comment (rbharath) [inline devtools/travis-ci/test_format_code.sh]: Wouldn't this select for only files ending in .py that are in contrib? (The opposite of command we want)

Comment (lilleswing) [inline devtools/travis-ci/test_format_code.sh]: -v is grep inverse -- so it excludes everything that has ".*contrib/.*"

Comment (rbharath) [inline devtools/travis-ci/test_format_code.sh]: Oh, yes you're right. Nonintuitive flags...

**Comment by rbharath** [comment]
LGTM

**Pull request merged**